test/install.c

```diff
@@ -1124,6 +1141,20 @@ static void install_test_bundle(InstallFixture *fixture,
 	args->cleanup(args);
 }
 
+static void install_test_bundle_twice(InstallFixture *fixture,
+		gconstpointer user_data)
+{
+	/* First run */
+	install_test_bundle(fixture, user_data);
+
+	/* Manual partial cleanup */
+	test_umount(fixture->tmpdir, "mnt");
+	test_rm_tree(fixture->tmpdir, "mnt");
+
+	/* Second run */
+	install_test_bundle(fixture, user_data);
```

Comment:
Can we check that the install was skipped? Perhaps by looking at the slot status data?

Also we should test the inverse: that the bundle is installed again if install-same is true.

Reply:
I don't see any way to check this with manageable effort at the moment or in the context of this fix.

Since we use the same bundle for installing, the only way to detect if an installation happened is by having a look at the time stamps in the status file. And I am not sure if enough time is spent between calls.

Anyway, adding support for parsing the status file here is quite complex from the layer we operate on.

With the current check, at least the older issue would have been detected. The other would have not.